packages/termux-tools/login

```diff
@@ -2,6 +2,11 @@
 
 if [ $# = 0 ] && [ -f @TERMUX_PREFIX@/etc/motd ] && [ ! -f ~/.hushlogin ] && [ -z "$TERMUX_HUSHLOGIN" ]; then
 	cat @TERMUX_PREFIX@/etc/motd
+
+	# TERMUX_VERSION env variable has been exported since v0.107 and PATH was being set to following value in <0.104. Last playstore version was v0.101.
+	if [ -f @TERMUX_PREFIX@/etc/motd-playstore ] && [ -z "$TERMUX_VERSION" ] && [ "$PATH" = "@TERMUX_PREFIX@/bin:@TERMUX_PREFIX@/bin/applets" ]; then
+		printf '\033[0;31m'; cat @TERMUX_PREFIX@/etc/motd-playstore; printf '\033[0m'
+	fi
```

Comment:
Should probably be outside `if [ ! -f ~/.hushlogin ] && [ -z "$TERMUX_HUSHLOGIN" ]`, we want to force people to see the message and really encourage them to move to a more up to date version (even if they have previously silenced the motd)

Reply:
Yeah that makes sense. I will do it.